packages/cli-kit/src/api.ts

```diff
@@ -2,5 +2,6 @@ import * as admin from './api/admin.js'
 import * as partners from './api/partners.js'
 import * as graphql from './api/graphql/index.js'
 import * as identity from './api/identity.js'
+import * as common from './api/common.js'
```

Comment:
I'd not export the entire module because it was designed to be internal. What about only exporting the function that you are using, `buildHeaders`?